src/functions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+function getAllGames(object $db): array {
+    $query = $db->prepare('SELECT * FROM `videogames`;');
```

Comment:
We don't like `SELECT *` in production. While `SELECT *` is fine for discovering which fields are available, it has performance impacts from how it does that. Select the fields you want to display manually please.

Reply:
Done :) Please can you review?